sdk/keyvault/keyvault-keys/test/CRUD.test.ts

```diff
@@ -41,7 +41,8 @@ describe("Keys client - create, read, update and delete operations", () => {
 
   // If this test is not skipped in the browser's playback, no other test will be played back.
   // This is a bug related to the browser features of the recorder.
-  if (isNode && !isPlayingBack) { // On playback mode, the tests happen too fast for the timeout to work
+  if (isNode && !isPlayingBack) {
+    // On playback mode, the tests happen too fast for the timeout to work
```

Comment:
do we need this comment in so many places? seems redundant

Reply:
I added this comment because we've run into issues from conditionally running these tests. I don't like that we're doing this, so I want to make sure we have some form of flag that annoys us into fixing this after the release.